meta-mender-overo-sdcard/recipes-bsp/u-boot/files/fw_env.config.default

```diff
@@ -0,0 +1,8 @@
+# Configuration file for fw_(printenv/saveenv) utility.
+# Up to two entries are valid, in this case the redundant
+# environment sector is assumed present.
+# Notice, that the "Number of sectors" is ignored on NOR.
+
+# MTD device name       Device offset   Env. size       Flash sector size       Number of sectors
+/dev/mtd2               0x00000          0x20000         0x20000                 1
+/dev/mtd2               0x20000          0x20000         0x20000                 1
```

Comment:
You set `CONFIG_ENV_IS_IN_MMC` in the U-boot patch, does this mean that this file is not used?

Reply:
It's probably just something that didn't get fully fleshed out when I switched to UBI based integration.  I can drop but if you guys prefer but I thought it would be good to keep it for future.  Thoughts?